src/components/profile/Profile.vue

```diff
@@ -193,6 +193,8 @@
       :empty="!sections.keys"
       message="No keys have been added yet."
       messageOwn="You haven't added any keys yet."
+      messageNoLdap="You need an LDAP account to add keys."
```

Comment:
I'd opt for a different language. _Need_ sounds like you're missing out.

Reply:
"Without an LDAP account you can't add keys."?

"You can't yet add keys without an LDAP account."?

Reply:
_Editing keys without an LDAP account not supported yet._?